packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -255,43 +289,44 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
             <div className="flex flex-col items-start space-y-1">
               {/* TODO: anchor shouldn't be nested within a button */}
               {isDepositMode ? (
-                <a
-                  href={`${getExplorerUrl(l1Network.id)}/token/${
-                    token.address
-                  }`}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className="text-xs text-blue-link underline"
-                  onClick={e => e.stopPropagation()}
-                >
-                  {shortenAddress(token.address).toLowerCase()}
-                </a>
+                <>
```

Comment:
Doesn't look like we need a fragment here (same before `tokenHasL2Address` below)

Reply:
We do if we want the nested ternary

Reply:
<img width="444" alt="Screenshot 2023-06-08 at 11 40 37" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/e970e3f3-5bd0-4dd3-bd60-9ad89c1f9571">

Reply:
now that I look at it, it just looks more confusing w/o Fragments anyway

Reply:
I've always liked having an extra ternary because it adds some additional indent and makes easier to read, but I can remove